src/airport.js

```diff
@@ -1,6 +1,44 @@
+const Plane = require(`./Plane.js`);
+
 class Airport {
-  // here's a starting point for you
 
-}
+    constructor(maximumCapacity = 10) {
+        this.airportPlanes = [];
+        this.maximumCapacity = maximumCapacity;
+    };
+
+    landPlane = plane => {
+        if (this.airportPlanes.length >= this.maximumCapacity) {
+            return this.airportPlanes.length;
+        }
+        else if (plane instanceof Plane && !this.planeAtAirport(plane)) {
+            this.airportPlanes.push(plane);
+        };
+
+    };
+
+    takeoffPlane = plane => {
+        const indexOfPlaneInAirportPlanes = this.airportPlanes.findIndex(planeManifest => planeManifest.id === plane.id);
+
+        if (indexOfPlaneInAirportPlanes > -1 && this.planeAtAirport(plane)) {
+            this.airportPlanes.splice(indexOfPlaneInAirportPlanes, 1);
+        };
+    };
+
+    setCapacity = (input) => {
+        this.maximumCapacity = input;
+    };
+
+    planeAtAirport = plane => {
+        if (this.airportPlanes.includes(plane)) {
+            return true;
+        }
+        else {
+            return false;
```

Comment:
This applies to many of the methods but I think just return a true/false boolean could be improved upon. Perhaps catching errors and returning a message to the user when invalid values or when a plane is already landed at the airport or when a plane that doesn't exist at the airport tries to take off.